javascriptv3/example_code/bedrock-agent/actions/create-agent.js

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fileURLToPath } from 'url';
+import { checkForPlaceholders } from "../lib/utils.js";
+
+import { BedrockAgentClient, CreateAgentCommand } from '@aws-sdk/client-bedrock-agent';
```

Comment:
Single vs double quotes. Let me know if you need any help with the git hook.

Reply:
If you already have the hook set up, you might have to run prettier manually on all your files.
`npx prettier -w javascriptv3/example_code/bedrock-agent javascriptv3/example_code/libs/utils`